controller/slot_filling.py

```diff
@@ -1,33 +1,50 @@
 from __future__ import print_function
-from flask import Flask, request, make_response, jsonify, session, Blueprint
 import sys
 import os
 import json
 import dialogflow_v2
 import requests
-from dialogflow_v2 import types
+import time
+import re
+import dateparser
+import dateutil.relativedelta
+import jsonpickle
 
+from flask import Flask, request, make_response, jsonify, session, Blueprint
+from dialogflow_v2 import types
 from google.cloud import language_v1, language
 from google.cloud.language_v1 import enums, types
 from text2digits import text2digits
-import time
-import dateparser
-import dateutil.relativedelta
 from datetime import datetime, date, time, timedelta
 from pprint import pprint
-
 from controller.accounting_head import sendResponse, getTags
-import re
+from controller.messages import *
+from datetime import datetime
+from sqlalchemy import create_engine, MetaData, Table, Column, select, insert, and_, update
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+engine = create_engine(os.getenv('SQLALCHEMY_SERVER_URL'))
+connection = engine.connect()
+metadata = MetaData()
+twilioKey = Table('whatsapp_company_twilio_accounts', metadata,
+                  autoload=True, autoload_with=engine)
+
+sessionVariable = Table('whatsapp_user_active_sessions', metadata,
+                        autoload=True, autoload_with=engine)
 
 slot_fill = Blueprint('slot_fill', __name__)
 
-os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = r"../intent.json"
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = os.getenv(
+    'SLOT_DIALOGFLOW_LOCATION')
 
 client = dialogflow_v2.SessionsClient()
-session = client.session_path('classify-intents-ujpxuu', '1234abcdpqrs')
+session = client.session_path(
+    os.getenv('SLOT_DIALOGFLOW_PROJECT_ID'), '1234abcdpqrs')
```

Comment:
what is the string '1234abcdpqrs' used for? Can we use a random string generator?

Reply:
Done. Replaced with random string generator